.github/workflows/publish.yml

```diff
@@ -58,6 +58,22 @@ jobs:
           delete-branch: true
           branch: bump-engine
 
+  test-engine-publish:
```

Comment:
this step is also defined in the `test-publish` file. Is this here intentionally also? IIUC this will run the same step twice in PR's.

Reply:
Nope was just fixing that :-) I originally just had it in publish.yml but realized it didn't run in this PR because we configured publish to only trigger for certain paths in PRs. Cleaner to just move it out to its own file, I just forgot to delete the original one.

Reply:
Fixed